tutorwordpress/patches/local-docker-compose-jobs-services

```diff
@@ -0,0 +1,7 @@
+wordpress-job:
+  image: {{ WORDPRESS_DOCKER_IMAGE }}
+  volumes:
+    - ../../data/wordpress:/var/www/html/wp-content
```

Comment:
I suggest using a sub-directory of wordpress: `../../data/wordpress/wp-content`. This will make it easier to add some other directories for the same plugin in the future.

Reply:
Agree